docs/user/production-considerations/task-manager-troubleshooting.asciidoc

```diff
@@ -681,32 +809,133 @@ Suppose the output of `stats.workload.value` looked something like this:
 
-These rough calculations give you a lower bound to the required throughput, which is _at least_ 440 tasks per minute to ensure recurring tasks are executed, at their scheduled time. This throughput doesn't account for nonrecurring tasks that might have been scheduled, nor does it account for tasks (recurring or otherwise) that might be scheduled in the future.
+These rough calculations give you a lower bound to the required throughput, which is _at least_ 410 tasks per minute to ensure recurring tasks are executed, at their scheduled time. This throughput doesn't account for nonrecurring tasks that might have been scheduled, nor does it account for tasks (recurring or otherwise) that might be scheduled in the future.
 
 Given these inferred attributes, it would be safe to assume that a single {kib} instance with default settings **would not** provide the required throughput. It is possible that scaling horizontally by adding a couple more {kib} instances will.
 
 For details on scaling Task Manager, see <<task-manager-scaling-guidance>>.
 
+
+[[task-manager-health-evaluate-the-capacity-estimation]]
+===== Evaluate the Capacity Estimation
+
+Task Manager is constantly evaluating its runtime operations and workload. This enables Task Manager to make rough estimates about the sufficiency of its capacity.
+
+As the name suggests, these are estimates based on historical data and should not be used as predictions. These estimations should be evaluated along side the detailed <<task-manager-health-monitoring>> stats before making changes to infrastructure. These estimations assumes all {kib} instances are configured identically.
+
+We recommend using these estimations when following the Task Manager <<task-manager-scaling-guidance>>.
+
+Evaluating the health stats above you can see the following output under `stats.capacity_estimation.value`:
+
+[source,json]
+--------------------------------------------------
+{
+  "observed": {
+    "observed_kibana_instances": 1, # <1>
+    "minutes_to_drain_overdue": 1, # <2>
+    "max_throughput_per_minute_per_kibana": 200,
+    "max_throughput_per_minute": 200, # <3>
+    "avg_recurring_required_throughput_per_minute": 28, # <4>
+    "avg_recurring_required_throughput_per_minute_per_kibana": 28,
+    "avg_required_throughput_per_minute": 28, # <5>
+    "avg_required_throughput_per_minute_per_kibana": 28
+  },
+  "proposed": {
+    "min_required_kibana": 1, # <6>
+    "proposed_kibana": 1, # <7>
+    "avg_recurring_required_throughput_per_minute_per_kibana": 28,
+    "avg_required_throughput_per_minute_per_kibana": 28
+  }
+}
+--------------------------------------------------
+<1> These estimates assume that there is one {kib} instance actively executing tasks
+<2> Based on past throughput the overdue tasks in the system could be executed within 1 minute
```

Comment:
Is there a guideline as to how many `minutes_to_drain_overdue` is too many? Comparing this example to the below example of an overloaded deployment, `1` minute to drain vs `12` minutes to drain doesn't seem too bad?

Reply:
Not really.
In theory, we shouldn't have any overdue tasks, so anything > 0 is a sign that we are low on capacity.
12 minutes isn't too bad until you realized that during those 12 minutes, additional overdue tasks will build up.

I tried to calrify that in the docs saying:
```
* There appear to be so many overdue tasks that it would take 12 minutes of executions to catch up with that backlog. This does not take into account tasks that might become overdue during those 12 minutes, so while this congestion might be temporary, the system could also remain consistently under provisioned and might never drain the backlog entirely.
```